### PR TITLE
Remove json-schema.org from configs

### PR DESCRIPTION
**Pull request by nickbp**
Let's see how this does...

`find . -iname *.json | xargs sed -i '/json-schema.org/d'`

### CODE REVIEW

File: repo/meta/schema/build-definition-schema.json [hunk 1]
```diff
@@ -198,7 +197,6 @@
 
 
     "config": {
-      "$ref": "http://json-schema.org/draft-04/schema#"
```
Comment (elezar) [inline repo/meta/schema/build-definition-schema.json:201]: Is an empty `config` object allowed?

File: repo/meta/schema/config-schema.json [hunk 2]
```diff
@@ -1,6 +1,4 @@
 {
-    "id": "http://json-schema.org/draft-04/schema#",
```
Comment (elezar) [inline repo/meta/schema/config-schema.json:2]: This is not a reference to a schema, but a field in the JSON. Are there any other implications to changing this?

File: repo/meta/schema/v3-repo-schema.json [hunk 3]
```diff
@@ -253,7 +252,6 @@
 
 
     "config": {
-      "$ref": "http://json-schema.org/draft-04/schema#"
```
Comment (elezar) [inline repo/meta/schema/v3-repo-schema.json:256]: Another empty `config` object.

**Review by takirala — APPROVED** [review]
LGTM

**Review by elezar — APPROVED** [review]
Let's hope this works @nickbp.

It may be good to link a ticket or description of the reason for this change.

There are a couple of places where the replacements are "non-standard" and I don't have enough context to know if these may cause problems.

If they are OK, then this looks good from my side.

As a note, we don't specify the schema in any of our own SDK-based packages.

File: repo/meta/schema/build-definition-schema.json (hunk 1 above)
Comment (nickbp) [reply repo/meta/schema/build-definition-schema.json:201]: Seems to be... It's not clear why the schema needed to be referenced again like this, given that it's already referenced at the root level.

File: repo/meta/schema/config-schema.json (hunk 2 above)
Comment (nickbp) [reply repo/meta/schema/config-schema.json:2]: Looks like someone messed up. This field is supposed to be a pointer to _your_ schema URL:

https://spacetelescope.github.io/understanding-json-schema/basics.html
```
Declaring a unique identifier
It is also best practice to include an id property as a unique identifier for each schema. For now, just set it to a URL at a domain you control, for example:

{ "id": "http://yourdomain.com/schemas/myschema.json" }
```

**Pull request merged**